XenAdmin/Controls/CustomDataGraph/ArchiveMaintainer.cs

```diff
@@ -169,7 +180,8 @@ private void Update()
             {
                 var xenObject = XenObject;
 
-                var serverWas = ServerNow; // get time before updating so we don't miss any 5 second updates if getting the past data
+                var serverWas =
+                    ServerNow; // get time before updating so we don't miss any 5 second updates if getting the past data

```

Comment:
Line breaks like this make the code less readable. I'd suggest moving ServerNow back to the previous line and move the commnet underneath.

Reply:
agreed but it looks like it's been addressed in other commits in this PR 👍